File: daily/双指针/双指针easy/26. 删除排序数组中的重复项.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    var sum=0;
    for(let i=0;i<nums.length;i++)
        if(nums[i+1]!=nums[i])
            nums[sum++]=nums[i];
     return sum;
};

File: daily/C++版本/26. 删除排序数组中的重复项.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int sum=0;
        nums.push_back(-9999);   //为了让原向量最后一个数能加到sum里面去
        for(int i=1;i<nums.size();i++)
            if(nums[i]!=nums[i-1])
                nums[sum++]=nums[i-1];
        return sum;   
    }
};
